detectors/node/opentelemetry-resource-detector-alibaba-cloud/test/detectors/AlibabaCloudEcsDetector.test.ts

```diff
@@ -84,56 +85,47 @@ describe('alibabaCloudEcsDetector', () => {
   });
 
   describe('with unsuccessful request', () => {
-    it('should throw when receiving error response code', async () => {
-      const expectedError = new Error('Failed to load page, status code: 404');
+    it('should return empty resource when receiving error response code', async () => {
```

Comment:
note to reviewer: the detector does not throw but rather returns a resource with no attribs.